src/simpla-img.html

```diff
@@ -188,12 +199,48 @@
         return { src: this.src, alt: this.alt };
       },
 
-      _tapHandler() {
+      /**
+       * Opens editor on tap if image is editable
+       * @return {undefined}
+       */
+      _goActiveIfEditable() {
         if (this.editable) {
           this.active = true;
         }
       },
 
+      /**
+       * Stops blur events on image when opening editor,
+       * to maintain illusion that image itself is editable
+       * @param  {Event} event Blur and Focusout events
+       * @return {undefined}
+       */
+      _maintainFocusWhenActive(event) {
+        let editor = document.querySelector(EDITOR_COMPONENT);
+
+        if (event.relatedTarget === editor) {
```

Comment:
I'd also check for if the `relatedTarget` is inside the editor, in case of Shady DOM e.g.

```js
if (event.relatedTarget === editor || editor.contains(event.relatedTarget))
```

though not 100% sure if it's needed in Shady DOM.